.github/workflows/deployment.yml

```diff
@@ -39,7 +39,7 @@ on:
 env:
   PRODUCT: apim
   ENVIRONMENT: ${{ github.ref_name }}
-  TIMEZONE: "Europe/London"
+  timezone: ${{ vars.TIMEZONE }}
```

Comment:
Why the change to lowercase?

Reply:
This is to keep it consistent with other repositories.